apps/epic-web/src/templates/workshop-template.tsx

```diff
@@ -432,21 +432,34 @@ const Header: React.FC<{module: Module; canView: boolean}> = ({
   )
 }
 
-const CourseMeta = ({
-  title,
-  description,
-}: {
-  title: string
-  description?: string | null | undefined
-}) => (
+const CourseMeta = ({product}: {product: SanityProduct}) => (
   <CourseJsonLd
-    courseName={title}
-    description={description || ''}
+    courseName={product.title || ''}
+    description={product.description || ''}
```

Comment:
we should beef these up. Crawlers expect certain char lengths for title and description, this goes for where ever normal SEO is being set as well (could be a different PR?) 

![image](https://github.com/skillrecordings/products/assets/6188161/50b9c9e2-1b61-40ee-be96-289484bb5007)